PokeAlarm/Utilities/PvpUtils.py

```diff
@@ -12,7 +12,7 @@ def calculate_cp(monster, atk, de, sta, lvl):
     cp = ((base_stats["attack"] + atk) * sqrt(base_stats["defense"] + de) *
           sqrt(base_stats["stamina"] + sta) * (multipliers[str(lvl)]**2)
           / 10)
-    return int(cp)
+    return int(cp) if int(cp) > 10 else 10
```

Comment:
Seems like we should be determining what can cause the formula above to result in a CP < 10 as 10 is the base CP for a large majority of Mon. Is the formula wrong or does Niantic abandon the formula for some CP 10 Mon?

Reply:
The cp formula is correct, but there's a min cp, especially for level 1 mons. Based on gamepress: `If CP would be less than 10, it is displayed as 10 instead`